### PR TITLE
docs: wikipedia yaml comments

### PR DESCRIPTION
**Pull request by jakob1111996**
Relates to #445 

I have added comments to the yaml files in the wikipedia-sentences example.
The example is still based on jina 1.2.0.

### CODE REVIEW

**Review by theUnkownName — APPROVED** [review]
LGTM

File: wikipedia-sentences/flows/index.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-!Flow
+!Flow                             # Define the flow used for indexing in this file
 version: '1'
-pods:
-  - name: encoder
-    uses: pods/encode.yml
+pods:                             # All pods that are executed in this flow are listed here
+  - name: encoder                 # Name for the encoder pod
```
Comment (FionnD) [inline wikipedia-sentences/flows/index.yml:4]: Assigned name right? The user can call this whatever they want. Perhaps we make that clear?

File: wikipedia-sentences/pods/encode.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-!TransformerTorchEncoder
-with:
-  pooling_strategy: auto
-  pretrained_model_name_or_path: distilbert-base-cased
-  max_length: 192
+!TransformerTorchEncoder                                  # Use a pytorch transformer from huggingface - expects an array of strings with size B and transforms it into an array of size B x D
```
Comment (FionnD) [inline wikipedia-sentences/pods/encode.yml:1]: Perbuilt by the Jina Team and accessible from hub.jina.ai right?

File: wikipedia-sentences/pods/index.yml [hunk 3]
```diff
@@ -1,16 +1,16 @@
-!CompoundIndexer
-components:
-  - !NumpyIndexer
-    with:
-      index_filename: vec.gz
-      metric: cosine
+!CompoundIndexer                      # Indexer that uses two indexers internally - one vector indexer to index the vector and one key-value indexer to index the meta information
```
Comment (FionnD) [inline wikipedia-sentences/pods/index.yml:1]: index the vector**s**

**Review by FionnD — APPROVED** [review]
LGTM, some minor comments.

**Pull request merged**